setup.cfg

```diff
@@ -12,3 +12,4 @@ warn_unused_configs = True
 #disallow_any_unimported = True
 disallow_untyped_defs = True
 disallow_incomplete_defs = True
+mypy_path = bindings/python/install_data
```

Comment:
will this be included in the package?

Reply:
This config is only picked up when running `mypy` directly in the repo, or via `pre-commit`.

The python packaging includes `bindings/python/install_data` such that when libtorrent is installed, `mypy` would find the type stubs automatically.